internal/kafka/command_topic_describe_onprem.go

```diff
@@ -15,15 +15,15 @@ import (
 )
 
 type PartitionData struct {
-	PartitionId int32   `human:"Partition" json:"partition" yaml:"partition"`
-	Leader      int32   `human:"Leader" json:"leader" yaml:"leader"`
-	Replicas    []int32 `human:"Replicas" json:"replicas" yaml:"replicas"`
-	ISR         []int32 `human:"ISR" json:"isr" yaml:"isr"`
+	Partition int32   `human:"Partition" serialized:"partition"`
+	Leader    int32   `human:"Leader" serialized:"leader"`
+	Replicas  []int32 `human:"Replicas" serialized:"replicas"`
+	Isr       []int32 `human:"ISR" serialized:"isr"`
 }
 
 type describeOutOnPrem struct {
-	topicOut
-	Partitions []*PartitionData `json:"partitions" yaml:"partitions"`
+	TopicOut
+	Partitions []*PartitionData `serialized:"partitions"`
```

Comment:
```suggestion
	Partitions []*PartitionData `human:"Partitions" serialized:"partitions"`
```

Reply:
No need; the combined struct is only used for serialized output.

Reply:
Why isn't it used for human output?

Reply:
We're combining a table and a list, and both table and list produce mangled human output here. So for non-serialized output options, I currently print the table and list separately.